### ISSUE TITLE
Remove System.Drawing dead code

### ISSUE THREAD

**Issue by danmosemsft** [dotnet/corefx#21296]
Attached is a zip containing removed.htm which shows everything ILLink thinks is dead and all.htm which shows everything, with removed in red. Everything red should be safe to delete.

[removed.zip](https://github.com/dotnet/corefx/files/1090302/removed.zip)

@mellinoe @hughbe do you think it is safe for folks to do this without tests?

**Comment by hughbe** [comment]
Most of this is interop code - compilation will fail if something goes wrong.

Other stuff are private ctors in a sealed class which can't be removed
```
  public sealed class SystemColors {
    private SystemColors();
  }
  public sealed class SystemFonts {
    private SystemFonts();
  }

```

**Comment by danmosemsft** [comment]
Yeah, that amongst others are exceptions I listed in this comment. Constants is another.
https://github.com/dotnet/corefx/issues/17905#issue-219402729
Of the rest it looks like a bunch is debug only and i did the scan in release.
I do see a little truly dead though.

**Comment by danmosemsft** [comment]
[removed.debug.zip](https://github.com/dotnet/corefx/files/1090336/removed.debug.zip)
Debug build .. a bit more helpful.

**Comment by mellinoe** [comment]
I have several other "cleanup" items listed here: https://github.com/dotnet/corefx/issues/20706

We may want to do some of the reorganization / refactoring / consolidation first, since that is likely to affect which parts of the library end up being dead.

**Comment by stephentoub** [comment]
> We may want to do some of the reorganization / refactoring / consolidation first

Yes, please.
